### ISSUE TITLE
Issue with public key sent by the server

### ISSUE THREAD

**Issue by lucasgalton** [yovanoc/cookie#2]
Hi, 

I tried the code from master today, and there's a problem when hitting the connect button. 
Logs from the js console: 

> Uncaught Error: error:0D0680A8:asn1 encoding routines:ASN1_CHECK_TLEN:wrong tag
    at Object.publicEncrypt (crypto.js:388)
    at RSAEncrypt.encryptStringWithRsaPublicKey (RSAEncrypt.ts?ddf8:25)
    at RSAEncrypt.cipherRSA (RSAEncrypt.ts?ddf8:55)
    at OnHelloConnectMessage (BasicFrame.ts?7188:31)
    at Dispatcher.emit (Dispatcher.ts?44b3:23)
    at Socket.socket.on (Network.ts?cb6f:64)
    at emitOne (events.js:115)
    at Socket.emit (events.js:210)
    at addChunk (_stream_readable.js:252)
    at readableAddChunk (_stream_readable.js:239)

The public key received from the server:

> -----BEGIN PUBLIC KEY-----
NzksODksNzUsLTMxLC02OSwxMTIsLTUzLDY2LC04LC01NSwtNSwtMzYsLTY0LC04MCwxMDMsMTE0LC00MSwtMjMsLTg5LDg1LDk3LC03Myw5OSwxMDQsODIsLTgzLC0xNCwtMjEsOTAsODAsODQsMTI3LC0yMywtMjQsLTU4LC0xMjUsMzMsNzYsLTg3LC05OSw4NSwwLDU0LC0xMTcsLTYyLDk0LDExNywtNTksLTU1LDExOSw2OSwtNDcsNjQsODMsNjMsMjksLTE4LC05NSw0NiwzNyw2MiwtMTIyLC03Niw1MSwtOTIsMTEyLDEyNCwtMzQsLTc1LDc3LC0xMiwxMDYsLTU4LC05OCwtMTIsNSwyNSwtMjUsLTEyMyw2MSw5MSw4OSwzMiwzMCwtMTQsLTU1LDEzLC01OCwtMTEsLTQyLC0xNSwtODUsMjUsLTE0LDgxLC0xMTgsNzksMzksLTEwMiw0OSwtNTQsLTEwLC05NCwzNiwtMTMsLTcxLDU1LC0yNSwtMTIyLC0yOCwxMDIsLTgsOTAsOTIsLTQxLC04NCw2LC0xMCwxNCwxNCwtNTUsMTEzLDQ1LC0xNSwzMCw1MiwtNTksLTgzLC01Miw5NywzNCwxOSw0MCwxMTQsLTMsLTExOCwxMTQsNTgsLTEyMyw3NiwtNTMsMywtMjksLTgsLTUsMTMsOTksMTI0LC0xMTksNTEsNjgsLTEyNCwtOTYsLTEwNCwtMTA1LC03NSw1OCwzOSwtMjgsOTYsMTIxLC05Niw1MywtNCwtOTUsLTM4LDI4LDExMCwtMywtODAsLTYyLDcsOSwtOTQsMTA5LC0xMTYsODcsLTgsLTMwLDQ3LC0xMjUsOCw3MCwtMTAxLDIxLDk3LC01MiwtMTE3LDEwNSw0MSwxMDYsLTQsLTEzLC0xMjMsMCwtNjEsNjMsLTM4LC04MywtNzAsMTA4LC0xMjQsMTA1LDYsMTE3LDMyLDYsNjYsMTE2LC0xMDcsLTg2LDkzLC00NCwtNTcsLTI4LDMwLC03NSw2Miw4MSw3NSw4NywtNTMsMTA2LC0xMDAsOTgsLTczLDEyNywtNzgsNDYsLTEwNSwxOSwtNCwtNzMsOTAsLTgzLDc2LC0xNiwzOSwtMTIyLC02NiwxMDAsLTUwLC0yOCwtNDIsLTI0LC0yOCw2OCwxMjUsMjAsOTgsLTQyLDExMCwtMTI3LC00MSwxMTQsMTUsMzQsMjMsLTU1LC0xMTEsLTEwMiwtMzgsLTY1LC00NCw2OSwtNDYsLTEyLDE0LDEwNCwtMTI0LC0xMjgsLTI4LC0xMDEsMTAwLDQ0LDEyNyw0OCwtMTcsLTg0LDgwLC0xMTYsMTA2LC0yNiwtODAsLTI1LDk1LDE2LDExNyw3NiwxMjAsMjksNjcsODAsLTEwMiw1OCwxOSwtMTAsLTg5LC01NywxMjAsMTgsLTYxLC04NywtODQsMzM=
-----END PUBLIC KEY-----

Any idea what might cause this ?
Thank you, best regards, 
Lucas

**Comment by yovanoc** [comment]
Yes this is because this project is very new, we have to do the credentials encryption at the connection
